pydarn/plotting/grid.py

```diff
@@ -0,0 +1,261 @@
+            parameter = "vector.{param}.median".format(param=parameter)
+
+        # Find the record corresponding to the start time
+        if start_time is not None:
+            for record in range(len(dmap_data)):
+                dtime = dt.datetime(dmap_data[record]['start.year'],
+                                    dmap_data[record]['start.month'],
+                                    dmap_data[record]['start.day'],
+                                    dmap_data[record]['start.hour'],
+                                    dmap_data[record]['start.minute'])
+                time_diff = dtime - start_time
+                if time_diff.seconds/60 <= time_delta:
+                    break
+            if time_diff.seconds/60 > time_delta:
+                raise plot_exceptions.NoDataFoundError(parameter,
+                                                       start_time=start_time)
+
+        _, aacgm_lons, _, _, ax = Fan.plot_fov(dmap_data[record]['stid'][0],
+                                               dtime, boundary=fov,
+                                               lowlat=lowlat)
+
+        data_lons = dmap_data[record]['vector.mlon']
+        data_lats = dmap_data[record]['vector.mlat']
+
+        # Hold the beam positions
+        shifted_mlts = aacgm_lons[0, 0] - \
+            (aacgmv2.convert_mlt(aacgm_lons[0, 0], dtime) * 15)
+        shifted_lons = data_lons - shifted_mlts
+        thetas = np.radians(shifted_lons)
+        rs = data_lats
+
+        # Colour table and max value selection depending on parameter plotted
+        # Load defaults if none given
+        if cmap is None:
+            cmap = {'vector.pwr.median': 'plasma',
+                    'vector.vel.median': cmocean.cm.haline_r,
+                    'vector.wdt.median': PyDARNColormaps.PYDARN_VIRIDIS}
+            cmap = plt.cm.get_cmap(cmap[parameter])
+
+        # Setting zmin and zmax
+        defaultzminmax = {'vector.pwr.median': [0, 50],
+                          'vector.vel.median': [0, 1000],
+                          'vector.wdt.median': [0, 250]}
+        if zmin is None:
+            zmin = defaultzminmax[parameter][0]
+        if zmax is None:
+            zmax = defaultzminmax[parameter][1]
+
+        norm = colors.Normalize
+        norm = norm(zmin, zmax)
+
+        data = dmap_data[record][parameter]
+
+        # Plot the magnitude of the parameter
+        ax.scatter(thetas, rs, c=data,
+                   s=2.0, vmin=zmin, vmax=zmax, zorder=5, cmap=cmap)
+
+        # If the parameter is velocity then plot the LOS vectors
+        if parameter == "vector.vel.median":
+
+            # Get the azimuths from the data
+            azm_v = dmap_data[record]['vector.kvect']
+
+            # Number of data points
+            num_pts = range(len(data))
+
+            # Angle to "rotate" each vector by to get into same reference frame
+            # Controlled by longitude, or "mltitude"
+            alpha = thetas
```

Comment:
That's fair, but I suppose it was to distinguish `alpha` as the rotation parameter, which happens to also be the same as `thetas`.

Reply:
Just make sure you're not doing anything to either one of the values later because depending on the type of copy you can end up changing both of them instead of making a copy and then only changing one.
I don't think you are though so it should be fine.

Reply:
because he is not storing anything in alpha it's not affecting thetas, but he could just call all `thetas` `alpha`. 

Why  `alpha`?

Reply:
I think it's the letter alpha in most maths books for this rotation. But we do use alpha in another file (for fan plots now?)  in another way, so we might want to choose something else instead someone forgets and we end up mixing this value with transparency.